ctapipe/io/simteleventsource.py

```diff
@@ -51,6 +53,10 @@ def build_camera_geometry(cam_settings, telescope):
 
 class SimTelEventSource(EventSource):
     skip_calibration_events = Bool(True, help='Skip calibration events').tag(config=True)
+    zcat = Bool(
```

Comment:
All good .. just one random thought.

is `zcat` maybe a bit too implicit as a parameter name? Could `only_forward_seeking` or `no_back_seek`  or so, be maybe a bit simpler to understand? 
My point is, that `zcat` is somehow an implementation detail ... and this might not be good as a parameter name here...

Reply:
Yes, I agree. 

I think `seekable` would be better, and then directly fail if it's not possible.